components/greetings/greetings.tsx

```diff
@@ -1,23 +1,24 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import styles from "./greetings.module.css";
 
-const date = new Date();
-const currentTime = date.getHours();
-
-let greetings;
+function Greetings() {
+  const [greeting, setGreeting] = useState(null);
+  useEffect(() => {
+    const date = new Date();
+    const currentTime = date.getHours();
```

Comment:
```suggestion
    const currentTime = new Date().getHours()
```

If you only need the current time you could shorten this a little ...

Reply:
Thank you, will do!

Reply:
> LGTM! 🎉 Do you have a heroku Deployment already? Then you can include the Link in your Pull Request, so we can look at the app! 🧐

Till now I have been deploying the main branch only :) I will sort it